src/borg/archiver.py

```diff
@@ -1233,11 +1233,13 @@ def do_break_lock(self, args, repository):
 
         {now}
 
-            The current local date and time.
+            The current local date and time, by default in ISO-8601 format.
+            You can also supply your own `format string <https://docs.python.org/3.5/library/datetime.html#strftime-and-strptime-behavior>`_, e.g. {now:%Y-%m-%d_%H:%M:%S}
```

Comment:
as 3.4 is currently our minimum requirement, it's maybe better to link to the 3.4 docs.
not sure if there was any change there in 3.5, so stuff there would not work on a py34 borg.